Fix approve transfer API

Hwesta opened the pull request:
MicroServiceChainChoice had been updated to have a foreign key to MicroServiceChain, but not all cases that used chainavailable were updated.  Fixed the last one.

Other cleanup while fixing this:
- Add instructions to API whitelist template specifying the format
- Whitelisted IPs for API are now whitespaces separated, instead of \r\n
- Clean up import list
- Add logging
- Add unicode function to MicroServiceChain & MicroServiceChainChoice

Supports artefactual/archivematica-storage-service#7

Hwesta committed:
Fix approve transfer API

refs #6583

MicroServiceChainChoice had been updated to have a foreign key to
MicroServiceChain which can be used directly, instead of fetching the
MicroServiceChain with the correct UUID.  Update approve_transfer for
this change.

mistydemeo commented on src/dashboard/src/components/api/views.py:
```diff
@@ -15,13 +15,26 @@
+from main import models
+
+logger = logging.getLogger(__name__)
+logging.basicConfig(filename="/tmp/archivematicaDashboard.log",
```
I wonder if we should wait on cleaning up the logging permissions before adding more logs that hardcode to `/tmp/archivematicaDashboard.log`? The more we do this, the less it becomes possible to run tests as anything other than the archivematica user, since simple imports can run into permissions issues.

Hwesta replied:
Since #34 exists now, whichever one gets merged first can be updated to fix the logging here.  Sound reasonable?

mistydemeo commented:
Looks good aside from my logging comment.

Hwesta committed:
Cleanup

Add instruction to API whitelist template, updated code to match.
Whitelisted IPs are now whitespace separated, instead of newline
separated.  Add display functions for MicroServiceChain and
MicroServiceChainChoice.  Cleanup imports.

Hwesta commented:
Removed logging module and config, but left the comment out logger.foo lines.
